drafts/celebrating-with-pictures.md

```diff
@@ -0,0 +1,81 @@
+# Celebrating a birthday with pictures
```

Comment:
You should include front matter like the title of the post and the type of page:

```yaml
---
layout: post
title: Towards Zero-Downtime Deployments
author: jelle
---
```